src/Halogen/HTML/CSS.purs

```diff
@@ -64,8 +65,8 @@ style =
   rights = concatMap $ foldMap singleton
 
 -- | Render a set of rules as a `style` element.
-stylesheet ∷ ∀ p i. CSS → HTML p i
+stylesheet ∷ ∀ p i. CSS → HC.HTML p i
```

Comment:
We can use [`HTMLstyle` from `DOM.HTML.Indexed`](https://github.com/slamdata/purescript-dom-indexed/blob/a27743fc798c58e67b87a80e790e272b04482c9e/src/DOM/HTML/Indexed.purs#L538-L544) for the `p` here.